src/controllers/update/payment.controller.ts

```diff
@@ -0,0 +1,31 @@
+import { Request, Response, NextFunction } from "express";
+import { createAndLogErrorRequest, logger } from "../../utils/logger";
+import * as config from "../../config";
+import { ApplicationData } from "model";
+import { getApplicationData } from "../../utils/application.data";
+import { OverseasEntityKey, PaymentKey } from "../../model/data.types.model";
+import { CreatePaymentRequest } from "@companieshouse/api-sdk-node/dist/services/payment";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
```

Comment:
Do we need/want a log message here at the start of the action like in our other controllers?

`logger.debugRequest(req, `${req.method} ${req.route.path}`);`